code/src/style.css

```diff
@@ -0,0 +1,151 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #191414;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+a {
+  text-decoration: none;
+}
+
+.heading {
+  color: white;
+  font-family: 'Helvetica Neue';
+  font-size: 18px;
+  margin: 40px auto 10px auto;
+  max-width: 300px;
+}
+
+hr {
+  width: 300px;
+  margin-bottom: 17px;
+  opacity: 0.2;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: 0 auto 50px auto;
+  justify-items: center;
+  width: 300px;
+  gap: 20px;
+}
+
+.song-card {
+  justify-content: center;
+}
+
+.album-art-container {
+  position: relative;
+}
+
+.title {
+  font-size: 14px;
+  font-family: 'Helvetica Neue';
+  color: white;
+  max-width: 300px;
+}
+
+.title:hover {
+  text-decoration: underline;
+}
+
+.artist {
+  font-size: 14px;
+  font-family: 'Helvetica Neue';
+  color: #a0a0a0;
+}
+
+.artist:hover {
+  text-decoration: underline;
+}
+
+.icons-overlay {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
```

Comment:
I don't think you need theese lines of code when I check the dev tools. They don't seem to do anything since you already saying that it should be position: absolute; top: 50%; left:50%; which centers it :)
![sarah Johanna](https://user-images.githubusercontent.com/81567468/137957890-a13f61e8-f7c5-4114-88d2-57277e35cba5.PNG)